Sources/HaCTML/Attributes.swift

```diff
@@ -1,13 +1,22 @@
 private func stringAttribute(_ key: String) -> AttributeKey<String> {
-  return .init(key, apply: { .text($0) })
+  return AttributeKey(key, apply: { .text($0) })
 }
 
 private func numberAttribute(_ key: String) -> AttributeKey<Int> {
-  return .init(key, apply: { .text(String($0)) })
+  return AttributeKey(key, apply: { .text(String($0)) })
 }
 
-private func CSSAttribute(_ key: String) -> AttributeKey<[String: String]> {
-  return .init(key, apply: { .text($0.map({ "\($0): \($1);" }).joined(separator: " ")) })
+private func CSSAttribute(_ key: String) -> AttributeKey<[String: String?]> {
```

Comment:
50/50 on whether this is a good idea. Fundamentally this attribute deals with Strings to Strings but having this value be optional made things easier at the point of use (for me)

Reply:
I share your sentiment. This should be a concern if an optional value can sneak its way in there unintentionally. Maybe we can come up with a friendly way to say "yeah this is optional but I'm doing it on purpose", so the behaviour is explicit?